devel/swiftformat/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        nicklockwood swiftformat 0.54.3
+github.tarball_from archive
+revision            0
+
+platforms           {darwin >= 21}
+categories          devel
+license             MIT
+maintainers         {jrabinow @jrabinow} openmaintainer
```

Comment:
the first entry here should be your (obfuscated) e-mail address; what you have now implies a ....@macports.org address which isn't correct

Reply:
Sorry about that. I went and looked up [the guide](https://guide.macports.org/#reference.keywords.maintainers) to make sure I got it right this time.

Judging by the output of `rg maintainers|grep -v nomaintainer|grep -E '\{@[^ ]*\}'`, there aren't many ports without email, and so providing an email address is preferable. I'm wondering if it's nevertheless accepted to not provide an email address

Reply:
Pre-github that was the only way - I don't know what the policy is now. I know we'd prefer a email, as we mostly discuss over mailing list.